### PR TITLE
#154080079 Add logic for Incident Filters

### PR DESCRIPTION
**Pull request by RayNjeri**
#### What does this PR do?
Adds functionality to Flag's and countries button.
#### Where should the reviewer start?
[#154080079](https://www.pivotaltracker.com/n/projects/2117172)

### CODE REVIEW

**Comment by RayNjeri** [comment]
@NaiRobley not exactly the PR was raised last week b4 the recent merges

**Comment by NaiRobley** [comment]
@RayNjeri did you rebase from `develop` or `staging`?

**Comment by kimobrian** [comment]
@RayNjeri Still seeing conflicts with staging

**Comment by kimobrian** [comment]
@RayNjeri Everything looks good but I was unable to test since it still uses the old `server.js`. Can you merge with staging and make necessary updates. Also remove the hard-coded urls in incidentAction and notesAction files and use the constants instead.

File: package.json [hunk 1]
```diff
@@ -95,8 +95,6 @@
         "style-loader": "^0.19.0",
         "webpack": "^3.10.0",
         "webpack-combine-loaders": "^2.0.3",
-        "webpack-dev-middleware": "^1.12.0",
```
Comment (kimobrian) [inline package.json]: @RayNjeri ???? Deleting these packages is breaking things

**Pull request merged**